packages/video_player_avplay/tizen/inc/plusplayer/plusplayer_wrapper.h

```diff
@@ -0,0 +1,534 @@
+#ifndef PLUSPLAYER_WRAPPER_H_
```

Comment:
1.we should add  below text at the beginning of file
"
// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
"
2. please change "#define PLUSPLAYER_WRAPPER_H_" to "#define FLUTTER_PLUGIN_PLUSPLAYER_WRAPPER_H"

Reply:
fixed